src/Core/Core.csproj

```diff
@@ -1,16 +1,35 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <RootNamespace>CodeMonkeys.Core</RootNamespace>
     <SynchReleaseVersion>false</SynchReleaseVersion>
     <ReleaseVersion>2.0</ReleaseVersion>
+    <AssemblyName>CodeMonkeys.Core</AssemblyName>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Version>2.0.0</Version>
```

Comment:
Do we really now want to start with 2.0? Because otherwise we could also start with 1.0 again because we have changed the package name